source/common/stats/isolated_store_impl.h

```diff
@@ -101,19 +102,37 @@ class IsolatedStoreImpl : public StoreImpl {
 
   // Stats::Scope
   Counter& counterFromStatName(StatName name) override { return counters_.get(name); }
+  Counter& counterFromStatName(StatName name, const StatNameTagVector& tags) override {
+    SymbolTable::StoragePtr suffixed_storage = TagUtility::addTagSuffix(name, tags, symbolTable());
```

Comment:
What you have is fine, but I have a minor suggestion and I'm fine if you take it or leave it.

You can make addTagSuffix be a non-static method, and have it store suffix_storage as a member var, and StatName can be the return value.

It won't reduce the lines of code at the call-sites, but might be easier to read. Up to you.